tests/test_wave_01.py

```diff
@@ -119,12 +119,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    for movie_dict in janes_data["watched"]:
+        assert MOVIE_TITLE_1 in movie_dict["title"]
```

Comment:
This works, but might be a bit overly complicated. We want to have as little logic in our tests as possible. Can you think of how you might achieve the same check without a loop?